cordformation/src/test/kotlin/net/corda/plugins/DockerformTest.kt

```diff
@@ -1,14 +1,28 @@
 package net.corda.plugins
 
 import org.assertj.core.api.Assertions.assertThat
+import org.gradle.internal.impldep.org.junit.Ignore
 import org.gradle.testkit.runner.TaskOutcome
 import org.junit.jupiter.api.Test
 
 class DockerformTest : BaseformTest() {
 
+
+//    @Test
```

Comment:
Let's not leave commented out tests. Either this should work and it should be uncommented (and potentially fixed) or it's an invalid test and it should be removed.

Reply:
Test has now been commented out